contracts/MasterChef.sol

```diff
@@ -233,13 +203,9 @@ contract MasterChef is Ownable {
         user.rewardDebt = 0;
     }
 
-    // Safe sashimi transfer function, just in case if rounding error causes pool to not have enough SASHIMIs.
-    function safeSashimiTransfer(address _to, uint256 _amount) internal {
-        uint256 sashimiBal = sashimi.balanceOf(address(this));
-        if (_amount > sashimiBal) {
-            sashimi.transfer(_to, sashimiBal);
-        } else {
-            sashimi.transfer(_to, _amount);
-        }
+    // Transfer ERC20 and update the required ERC20 to payout all rewards
+    function erc20Transfer(address _to, uint256 _amount) internal {
+        sashimi.transfer(_to, _amount);
```

Comment:
This token has been removed, but still used? Should be replaced by something else.